README.md

```diff
@@ -212,6 +212,14 @@ It is possible to interrupt the watcher from another thread with:
 watcher.finish
 ```
 
+#### Get a proxy URL
+You can get a complete URL for connecting a kubernetes entity via the proxy.
+
+```ruby
+client.proxy_url('services', 'srvname', '5001-tcp', 'ns')
```

Comment:
Yeah you're right just tested that here. Has to be name for service proxying (or no port at all if single port service).

Name doesn't work for pod proxying though, at least under the "new" location... Has to be no port or int. Name doesn't work.

Inconsistency ftw.